lib/SimpleSAML/Utilities.php

```diff
@@ -230,6 +230,10 @@ private static function _doRedirect($url, $parameters = array())
             SimpleSAML_Logger::warning('Redirecting to a URL longer than 2048 bytes.');
         }
 
+        if (isset($GLOBALS['SimpleSAML.debugDisableRedirect'])) {
+                throw new SimpleSAML_Error_Exception('Redirect was attempted with redirecting disabled: ' . var_export($url, TRUE));
+        }
+
```

Comment:
The `SimpleSAML_Utilities` class is deprecated and scheduled for removal. This check, in case it must be here inevitably, should be in `SimpleSAML_Utils_HTTP::redirect()` instead.